core/tests/test_utils.py

```diff
@@ -546,7 +660,12 @@ def save_new_valid_collection(
         return collection
 
     def publish_collection(self, owner_id, collection_id):
-        """Publish the collection of given collection_id."""
+        """Publish the collection with the given collection_id.
+
+        Args:
+            owner_id: str. user_id of the owner of the collection.
```

Comment:
I think you missed a previous comment: user_id --> The user_id.

In the future, please verify that all prior comments have been addressed before submitting.

Reply:
Sorry, I have missed that. Its done now!